src/js/veteran-representative/config/form.js

```diff
@@ -0,0 +1,274 @@
+              first: {
+                "ui:title": "Claimant’s first name",
+              },
+              last: {
+                "ui:title": "Claimant’s last name",
+              },
+              middle: {
+                "ui:title": "Claimant’s middle name",
+              },
+              suffix: {
+                "ui:title": "Claimant’s suffix",
+              },
+            }),
+            // FIX: need a default field for dropdown
+            relationship: {
+              "ui:select": "Relationship to Veteran",
+            },
+            claimantAddress: { "ui:title": "Address" },
+            claimantEmail: { "ui:title": "Email address" },
+            claimantDaytimePhone: phoneUI("Daytime phone number"),
+            claimantEveningPhone: phoneUI("Evening phone number"),
+            appointmentDate: currentOrPastDateUI(
+              "Date of claimaint's appointment"
+            ),
+          },
+          schema: {
+            type: "object",
+            required: [
+              'fullName',
+              'claimantAddress',
+              'claimantEmail',
+              'claimantDaytimePhone',
+              'claimantEveningPhone',
+              'relationship',
+              'appointmentDate',
+            ],
+            properties: {
+              fullName,
+              claimantAddress,
+              claimantEmail,
+              claimantDaytimePhone,
+              claimantEveningPhone,
+              relationship,
+              appointmentDate,
+            },
+          },
+        },
+      },
+    },
+    veteranServiceOrganization: {
+      title: "Veteran Service Organization",
+      pages: {
+        veteranServiceOrganization: {
+          path: "veteran-service-organization",
+          title: "Veteran Service Organization",
+          uiSchema: {
+            organizationName: { "ui:title": "Service organization name" },
+            organizationEmail: {
+              "ui:title": "Service organization email address",
+            },
+            organizationRepresentativeName: {
+              "ui:title": "Official representative's name",
+            },
+            organizationRepresentativeTitle: {
+              "ui:title": "Official representative's job title",
+            },
+          },
+          schema: {
+            type: "object",
+            required: [
+              'organizationName',
+              'organizationEmail',
+              'organizationRepresentativeName',
+              'organizationRepresentativeTitle',
+            ],
+            properties: {
+              organizationName,
+              organizationEmail,
+              organizationRepresentativeName,
+              organizationRepresentativeTitle,
+            },
+          },
+        },
+      },
+    },
+    authorizationForRepresentativeAccessToRecords: {
+      title:
+        "Authorization for Representative's Access to Records Protected by Section 7332, TITLE 38, U.S.C.",
```

Comment:
This is a very long title. More of a design issue, though.

Reply:
@carolynwill what do you suggest I shorten this to?

Reply:
Ooh yeah this one is a doozy. How about we cut everything from "protected...." So it reads "Authorization for Representative's Access to Records"